File: quan.java/quan-config/src/test/java/quan/config/CardIds.java
package quan.config;

/**
 * 卡片ID<br/>
 * 代码自动生成，请勿手动修改
 */
public enum CardIds {

    /**
     * 卡片1
     */
    card1,

    /**
     * 卡片2
     */
    card2;

    public int value() {
        return CardConfig.getByKey(name()).id;
    }

}

File: quan.lua/src/test/config/CardIds.lua
local CardConfig = require("test.config.CardConfig")

---
---卡片ID
---@author 代码自动生成，请勿手动修改
local CardIds = {}

---
---卡片1
---@return int
function CardIds.card1()
    return CardConfig.getByKey("card1").id
end

---
---卡片2
---@return int
function CardIds.card2()
    return CardConfig.getByKey("card2").id
end

---
---卡片4
---@return int
function CardIds.card4()
    return CardConfig.getByKey("card4").id
end

---
---卡片5
---@return int
function CardIds.card5()
    return CardConfig.getByKey("card5").id
end

return CardIds